Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-DM-3-A.yml

```diff
@@ -20,15 +20,15 @@ title:
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  Security defects are either resolved within a predefined time or compensating controls are introduced. 
+  Security defects are resolved (or mitigated) within a predefined time. 
```

Comment:
I saw the idea of compensating controls as quite important. What about:

"Security defects are resolved or mitigated (possibly also with compensating controls) within a predefined time."

That would also be 15 words (while that was not the priority).

Reply:
OK for me.